package.json

```diff
@@ -8,10 +8,10 @@
     "url": "https://github.com/refractproject/minim-api-description.git"
   },
   "scripts": {
-    "precoveralls": "peasant cover",
-    "coveralls": "coveralls <coverage/lcov.info",
+    "coveralls": "peasant cover && coveralls <coverage/lcov.info",
```

Comment:
Can we have `npm run cover && coveralls ...` here? We should rely `npm run cover` since `cover` can be changed.